packages/oruga-next/src/components/sidebar/Sidebar.vue

```diff
@@ -334,15 +335,18 @@ onBeforeUnmount(() => {
     }
 });
 
+let removeOutsideListener = null;
+
 /** add outside click event listener */
 function addHandler(): void {
-    if (isClient && !props.overlay)
-        setTimeout(() => document.addEventListener("click", clickedOutside));
+    if (isClient && !props.overlay) {
+        removeOutsideListener = useClickOutside(rootRef, clickedOutside);
```

Comment:
I think `rootRef` is wrong, you have to use `sidebarContent`.

Reply:
yeah i like that much better. done.